server/service/src/sync/translations/name.rs

```diff
@@ -230,12 +231,12 @@ impl SyncTranslation for NameTranslation {
                 .or(created_date.map(|date| date.and_hms_opt(0, 0, 0).unwrap())),
             date_of_death,
             custom_data_string,
+            is_active: existing_name.map_or(true, |name| name.is_active),
```

Comment:
OG doesn't have an `is_active` field, so this will always be true.

It does have an `is_deleted` field, but that doesn't actually get used so any merged names are deleted...

Reply:
Don't think its possible right now, but thought about some case where the name is soft-deleted locally and another UPSERT record is received...

However, as of https://github.com/msupply-foundation/open-msupply/pull/3693 I plan to change it to `deleted_datetime: Option<_>` which will solve it nicely as well.

Reply:
The issue is that the merged name is still viewable for me tho? And the is_active bool is true for the merged name